open-company/open-company-web
# Add section fixes

card: https://trello.com/c/ERcrMKS8

test:
- add a simple section
- check that the title has the proper text
- check that the body contains the proper text as placeholder
- check that the buttons CTA are DELETE and CREATE
- save the section
- add a finances section
- check the title and the notes body for the right text and placeholder
- check that the table is showing the last 12 months (from November 2015 to December 2014)
- delete the section (use the DELETE button at the bottom)
- check that the finances section is removed from the ToC and the list of sections
- add again the finances section
- add a couple of months of data
- click CREATE
- wait for the chart to appear
- check the chart data
- edit the finances section
- save the data
- check the chart again
- check the code
- check the CI outcome
- merge
